linux_os/guide/services/cron_and_at/service_cron_enabled/rule.yml

```diff
@@ -14,11 +14,15 @@ rationale: |-
 
 severity: medium
 
+identifiers:
+    cce@sle15: CCE-91437-4
```

Comment:
``` 
cce CCE-91437-4 is included in files: 
 - linux_os/guide/services/cron_and_at/service_cron_enabled/rule.yml
 - linux_os/guide/services/cron_and_at/service_crond_enabled/rule.ym
```
Please select a new CCE number for this rule.

Reply:
Thanks @Mab879 should be resolved in 6fb67b1